examples/src/chains/map_reduce_lcel.ts

```diff
@@ -0,0 +1,156 @@
+import { BaseCallbackConfig } from "langchain/callbacks";
+import {
+  collapseDocs,
+  splitListOfDocs,
+} from "langchain/chains/combine_documents/reduce";
+import { ChatOpenAI } from "langchain/chat_models/openai";
+import { Document } from "langchain/document";
+import { PromptTemplate } from "langchain/prompts";
+import { StringOutputParser } from "langchain/schema/output_parser";
+import { formatDocument } from "langchain/schema/prompt_template";
+import {
+  RunnableLambda,
+  RunnableMap,
+  RunnablePassthrough,
+  RunnableSequence,
+} from "langchain/schema/runnable";
+
+// Initialize the OpenAI model
+const model = new ChatOpenAI({});
+
+// Define prompt templates for document formatting, summarizing, collapsing, and combining
+const documentPrompt = PromptTemplate.fromTemplate("{pageContent}");
```

Comment:
This seems unnecessary? Can't we just use that previous util you added?

I don't think the metadata is relevant here

Reply:
This is using one of the utils I added `formatDocument`?  I could use `formatDocumentsAsString` but then we'd lose passing the "source" docs through